src/App.tsx

```diff
@@ -1,39 +1,59 @@
 import React from 'react';
 import './App.scss';
+import { Clock } from './Clock/Clock';
 
 function getRandomName(): string {
   const value = Date.now().toString().slice(-4);
 
   return `Clock-${value}`;
 }
 
-export const App: React.FC = () => {
-  const today = new Date();
-  let clockName = 'Clock-0';
-
-  // This code starts a timer
-  const timerId = window.setInterval(() => {
-    clockName = getRandomName();
-  }, 3300);
-
-  // this code stops the timer
-  window.clearInterval(timerId);
-
-  return (
-    <div className="App">
-      <h1>React clock</h1>
-
-      <div className="Clock">
-        <strong className="Clock__name">
-          {clockName}
-        </strong>
-
-        {' time is '}
+type State = {
+  clockName: string,
+  clockTrue: boolean,
+};
```

Comment:
Consider using a more descriptive name for 'clockTrue', such as 'isClockVisible'.